pages/campaigns/_id/_batch/index.vue

```diff
@@ -727,19 +778,6 @@ export default {
         }
       }
       this.canUserQualify = this.checkUserQualify()
```

Comment:
this line still calls the function checkUserQualify which doesn't exists anymore and overwrites a computed property. will definitaly cause an error, make sure to test these kind of things

Reply:
Fixed ✅